internal/load/README.md

```diff
@@ -192,7 +193,7 @@ type API struct {
 	Events            map[string]string `yaml:"events"`         // set as events
 	EventsOnly        bool              `yaml:"events_only"`    // only generate events
 	Merge             string            `yaml:"merge"`          // merge into another eventType
-	Joinkey           string            `yaml:"joinkey"`        // merge into another eventType
+	JoinKey           string            `yaml:"join_key"`       // merge into another eventType
```

Comment:
Doesn't this change break the existing config files using the old key? maybe we should add a comment in the changelog (just in case)

Reply:
This is just the documentation. It seems it was changed in the code, but not in this readme file

Reply:
oh, my bad. I didn't see both changes were made in a README 🤦‍♂